www/js/diary/list.js

```diff
@@ -39,15 +39,6 @@ angular.module('emission.main.diary.list',['ui-leaflet',
       readAndUpdateForDay($rootScope.barDetailDate);
       $rootScope.barDetail = false;
     };
-    if($rootScope.displayingIncident == true) {
```

Comment:
when you are testing, if you see weird issues with page loading, you may want to restore this.

Reply:
I see. Might be an issue created when merging with other branches previously